cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandController.java

```diff
@@ -2,12 +2,7 @@
 
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.TreeMap;
+import java.util.*;
```

Comment:
Are we okay with `*` imports?

Reply:
Changed back to the individual imports. 

This was done by IntelliJ. My IDE is configured using [this](https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/blob/main/cos-build-tools/src/main/resources/eclipse-format.xml) file, but apparently it doesn't include the [import order](https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/blob/main/cos-build-tools/src/main/resources/eclipse.importorder).

I have now installed [this plugin](https://github.com/krasa/EclipseCodeFormatter#instructions) in IntelliJ to be able to use the .importorder as well, but even then it generates different imports from the current ones. I'd suggest we find a better way to auto-organize those so it is never anymore a source of discussion.